bio/mehari/download-transcript-db/wrapper.py

```
@@ -0,0 +1,30 @@
+
+import re
+
+version_re = re.compile(r"\d+\.\d+\.\d+")
+
+log = snakemake.log_fmt_shell(stdout=True, stderr=True)
+
+version = snakemake.params["version"]
+if not version_re.fullmatch(version):
+    raise ValueError("version must have format MAJOR.MINOR.PATCH")
+
+build = snakemake.params["build"]
```

Ruff (0.11.9)

Undefined name `snakemake`

(F821)